### DISCUSSION TITLE
How do I verify spectral reflection value scale is 0-1.0 or 0-100.0 in CGATS?

### DISCUSSION THREAD

**Discussion by diuming** [colour-science/colour#1026] [General]
### Question

I get a spectral(380nm to 730nm) reflection values saved in CGATS file.
according to i1Profiler it can be saved as 0-1.0 or 0-100.0 value scale.
How do i verify the reflection value scale is  0-1.0 or 0-100.0?

If I known the maximum reflection value maybe can resolve this problem. if so.
For example: 
The maximum reflection value is 1.5 any value over it denote the value scale is 0-100.0.

Is there a defined reflection maximum value?

Could you give me any advice?

thanks

**Comment by thomasmansencal** [comment]
Hi @diuming,

I lost my access to the standard so I cannot verify if there is a formal indication on how this is meant to be done. I sent an email to ANSI so that they reinstate my download, I will keep you posted if I can find more details!

Cheers,

Thomas

**Comment by glenndavis52** [comment]
Hi @diuming ,

I don't know about i1Profiler, but programs in the Argyll CMS suite write
CGATS files (with extension `.sp`) with the keyword `SPECTRAL_NORM`, e.g.
    `SPECTRAL_NORM  "100.000000"`

I could not find any documentation on `SPECTRAL_NORM` on argyllcms.com,
but the source code file `xspect.h` has this comment:
    `double norm;   /* Normalising scale value, ie. 1, 100 etc. */`
    
The ISO 28178:2009 standard does not mention `SPECTRAL_NORM`, but does have these 2 lines:
    SPECTRAL_PCT - Reflectance/transmittance percentage 
    SPECTRAL_DEC - Reflectance/transmittance
    
Argyll CMS does not use these; what about i1Profiler ?

Hope this helps,
Glenn

**Reply by thomasmansencal** [reply]
Hi @diuming and @glenndavis52,

I have quickly checked the standard and the keyword does not exist unfortunately, nor there is any reference to a normalisation specification.

Cheers,

Thomas
